### ISSUE TITLE
xfs filesystem free diskspace 1.7/1.8

### ISSUE THREAD

**Issue by smira** [siderolabs/talos#10021]
On Talos 1.8.3 the xfs file system have lot bigger log

```
xfs_info /dev/sdb10
meta-data=/dev/sdb10             isize=512    agcount=4, agsize=65536 blks
         =                       sectsz=4096  attr=2, projid32bit=1
         =                       crc=1        finobt=1, sparse=1, rmapbt=1
         =                       reflink=1    bigtime=1 inobtcount=1 nrext64=1
data     =                       bsize=4096   blocks=262144, imaxpct=25
         =                       sunit=0      swidth=0 blks
naming   =version 2              bsize=4096   ascii-ci=0, ftype=1
log      =internal log           bsize=4096   blocks=59985, version=2
         =                       sectsz=4096  sunit=1 blks, lazy-count=1
realtime =none                   extsz=4096   blocks=0, rtextents=0
```

Versus a Talos 1.7.7

```
/# xfs_info /dev/sdb10
meta-data=/dev/sdb10             isize=512    agcount=4, agsize=61035 blks
         =                       sectsz=4096  attr=2, projid32bit=1
         =                       crc=1        finobt=1, sparse=1, rmapbt=0
         =                       reflink=1    bigtime=1 inobtcount=0 nrext64=0
data     =                       bsize=4096   blocks=244140, imaxpct=25
         =                       sunit=0      swidth=0 blks
naming   =version 2              bsize=4096   ascii-ci=0, ftype=1
log      =internal log           bsize=4096   blocks=1419, version=2
         =                       sectsz=4096  sunit=1 blks, lazy-count=1
realtime =none                   extsz=4096   blocks=0, rtextents=0
```

We can't rerproduce both of these

**Comment by smira** [comment]
Data point: such small log was only available with Talos < 1.2:

```
log      =internal log           bsize=4096   blocks=2560, version=2
```

**Comment by smira** [comment]
The xfs log size was increased in commit https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/commit/?id=cdfa467edd2d1863de067680b0a3ec4458e5ff4a (and there are reasons to do so)

This change went in xfsprogs 5.15.

* Talos 1.2 shipped xfsprogs 5.14.2
* Talos 1.3-1.7 shipped xfsprogs 5.18
* Talos 1.8 shipped xfsprogs 6.8
* Talos 1.9 shipped xfsprogs 6.11
